src/index.tsx

```diff
@@ -29,9 +29,12 @@ const AppcuesReactNative = NativeModules.AppcuesReactNative
 export function setup(
   accountID: string,
   applicationID: string,
-  options?: ReactNativeOptions
+  options?: ReactNativeOptions,
 ) {
-  AppcuesReactNative.setup(accountID, applicationID, options);
+  AppcuesReactNative.setup(accountID, applicationID, options, { 
+    _applicationFramework: 'react-native',
+    _applicationFrameworkVersion: require('react-native/package.json').version
+  });
```

Comment:
This seems the most basic way we _could_ do this. Alternatively, we could expose the `additionalAutoProperties` directly in the `ReactNativeOptions` that get passed in, allowing other things from the customer as well. Then, we could either add or create those if they don't exist, and supplement with our own values here. My initial thought was to keep it basic and just use it for internal use here - since they can send their own custom properties on each event as desired elsewhere.